jax/experimental/jax2tf/tests/shape_poly_test.py

```diff
@@ -1718,16 +1718,159 @@ def _make_harness(group_name: str, name: str,
                   poly_axes=[0, None, 0]),
 ]
 
+
+def _test_one_harness(tst: tf_test_util.JaxToTfTestCase, harness: Harness):
```

Comment:
This function was before ShapePolyPrimitivesTest.test_prim. Separate it out so that I can call it from ShapePolyVmapPrimitivesTest also